validation-test/compiler_crashers_2_fixed/0189-rdar47586626.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-typecheck-verify-swift
```

Comment:
Maybe put this test case in test/attr/attr_autoclosure.swift?

Reply:
Technically this was a crash that's why I added it to validation suite but I can move it to test/attr/attr_autoclosure.swift, no problem :)

Reply:
Done!